nebula.dme

```diff
@@ -3537,5 +3537,6 @@
 #include "maps\~mapsystem\maps_unit_testing.dm"
 #include "maps\~unit_tests\unit_testing.dm"
 #include "mods\_modpack.dm"
+#include "mods\persistence\game\objects\structures\numbered_signs.dm"
```

Comment:
This needs to be in the persistence modpack instead

Reply:
Oh, you're right! I didn't even notice I put it in nebula.dme ^^;

Reply:
fixed